tamr-config.yml

```diff
@@ -12,6 +12,7 @@ TAMR_CONNECTION_INFO_TYPE: hbase-site
 TAMR_HBASE_NAMESPACE: ${hbase_namespace}
 TAMR_HBASE_COMPRESSION: snappy
 TAMR_HBASE_CONFIG_URIS: s3://${tamr_data_bucket}/${hbase_config_path}hbase-site.xml
+TAMR_HBASE_EXTRA_URIS: s3://${tamr_data_bucket}/${hbase_config_path}hbase-env.sh
```

Comment:
These are the only uncommented lines in the hbase-env.sh:
```
export HBASE_MANAGES_ZK=false
export HBASE_ROOT_LOGGER=INFO,DRFA
export HBASE_SECURITY_LOGGER=INFO,DRFAS
export HBASE_REGIONSERVER_OPTS=-Xmx12288m
```

They all seem to be server facing to me.

What's the justification as to why the client would need these settings too?

Automated tests do not set this property, and  @daniosim 's recent internal deployments have not needed this either, so I'm not 💯  convinced that this is necessary.

Reply:
That said, I think this would be a benign change if we did choose to add it.

Just someone needs to know why it's necessary.

Reply:
I had at least one instance that wasn't working properly until I added this, but there was also a time component (maybe hbase wasn't fully up yet). I'm going to close this for the time being as internal testing doesn't seem to require it. I'll reopen it if I re-encounter the issue